interface_config.js

```diff
@@ -160,6 +160,7 @@ var interfaceConfig = {
     SHOW_CHROME_EXTENSION_BANNER: false,
 
     SHOW_DEEP_LINKING_IMAGE: false,
+    SHOW_DEEP_LINKING_LOGO: true,
```

Comment:
The default value should be in a way if people do not update interface_config to continue having the same behavior. 
With current value those that do not update config will not see it.

Reply:
:thinking: ok, I can do that.